v20.2/alter-column.md

```diff
@@ -59,6 +59,8 @@ For an example of `ALTER COLUMN TYPE`, see [Increase a column type's precision](
 
 ## Examples
 
```

Comment:
I think this example should be appended to the end of the examples provided, as changing the data type of two columns at once is not the most common use of `ALTER COLUMN`